### ISSUE TITLE
question about the ip_attn_len

### ISSUE THREAD

**Issue by Weizhi-Zhong** [RedAIGC/StoryMaker#37]
Hi, thanks for your excellent work. 
From the running process of infer.py, it seems that the ip_attn_len is 40.
In the train_storymaker.py, the default value of ip_attn_len is 60, and the two lines below
https://github.com/RedAIGC/StoryMaker/blob/a7ce766dc30753c126c14247989f8664daca0369/train_storymaker.py#L651
https://github.com/RedAIGC/StoryMaker/blob/a7ce766dc30753c126c14247989f8664daca0369/train_storymaker.py#L655C1-L655C17
have "attn_bg/60" and "attn/60".
Maybe these two lines should be "attn_bg/ip_attn_len" and "attn/ip_attn_len", respectively?

**Comment by RedAIGC** [comment]
Thanks for your attention. Since we use attention masks from 60 cross-attention layers, we divide the attn_mask by 60. It can be thought of as a hyperparameter in the training process.
